panel/src/components/Writer/Toolbar.vue

```diff
@@ -1,9 +1,12 @@
 <template>
   <div class="k-writer-toolbar">
-    <k-dropdown v-if="Object.keys(nodeButtons).length > 1 && activeNode" @mousedown.native.prevent>
+    <k-dropdown v-if="Object.keys(nodeButtons).length" @mousedown.native.prevent>
       <k-button
-        :icon="activeNode.icon"
-        class="k-writer-toolbar-button k-writer-toolbar-nodes"
+        :icon="activeNode.icon || 'wand'"
```

Comment:
Maybe we use https://getkirby.com/docs/reference/panel/icons/title as fallback here as well?

Reply:
I thought title, but it can be assumed that the active node is heading. That's why I wanted it to be a different icon.  May be `headline`?

Reply:
I like `headline`!